tests/data/setup/simple-setup.py

```diff
@@ -0,0 +1,23 @@
+#
```

Comment:
This should have the ORT copyright since it comes from there

Reply:
Done!